rusk-wallet/src/lib/prompt.rs

```diff
@@ -363,6 +364,12 @@ fn is_valid_addr(addr: &str) -> bool {
 
 fn check_valid_denom(num: Dusk, balance: Dusk) -> Result<(), String> {
     if num.is_finite() && num > 0.0 {
+        if num > MAX_CONVERTIBLE_DUSK {
+            return Err("Cannot convert in uDUSK".to_owned());
```

Comment:
This error message is user-facing. Can we try to make it a little bit more helpful?